modules/oc-mirror-enclave-support-about.adoc

```diff
@@ -19,4 +19,35 @@ Enclave support has the following benefits:
 
 * You can mirror content directly from the centralized internal registry to enclaves without restarting the mirroring process from the internet for each enclave.
 
-* You can minimize data transfer between network stages, so to ensure that a blob or image is transferred only once from one stage to another.
\ No newline at end of file
+* You can minimize data transfer between network stages, so to ensure that a blob or image is transferred only once from one stage to another.
+
+== How to mirror to an enclave
+
+image::445_OpenShift_Enclave_support_0724.png[Enclave Support]
+
+The above illustration outlines the flow for using the oc-mirror plugin in different environments, including environments with and without an internet connection.
```

Comment:
In [this section](https://redhat-documentation.github.io/supplementary-style-guide/#accessibility-visual-info) of the SSG, we're advised to avoid indicating direction such as above and below. In this case, you'd want to use "previous" instead. Also, "image" is a simpler word imo and the one used most often throughout our docs:
```suggestion
The previous image outlines the flow for using the oc-mirror plugin in different environments, including environments with and without an internet connection.
```